cmd/config_test.go

```diff
@@ -50,50 +51,125 @@ func TestPasswordConfig(t *testing.T) {
 }
 
 func TestTLSConfigLoad(t *testing.T) {
+	t.Parallel()
 	null := "/dev/null"
 	nonExistent := "[nonexistent]"
-	cert := "testdata/cert.pem"
-	key := "testdata/key.pem"
-	caCert := "testdata/minica.pem"
+	cert := "../test/hierarchy/int-e2.cert.pem"
+	key := "../test/hierarchy/int-e2.key.pem"
+	caCertOne := "../test/hierarchy/root-x1.cert.pem"
+	caCertMultiple := "../test/hierarchy/multiple-roots.cert.pem"
+	caCertDuplicate := "../test/hierarchy/duplicate-roots.cert.pem"
```

Comment:
These TLS credentials should exist in the `test/grpc-creds` directory and be generated using: https://github.com/letsencrypt/boulder/blob/main/test/grpc-creds/generate.sh